packages/cli/src/pull/handler.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  DeployConfig,
+  getProject,
+  getConfig,
+  getState ,
+} from '@openfn/deploy';
+import type { Logger } from '../util/logger';
+import { DeployOptions } from '../deploy/command';
+import path from 'path';
+import fs from 'node:fs/promises';
+
+
+
+
+async function pullHandler(
+  options: DeployOptions,
+  logger: Logger,
+) {
+  try {
+    const config = mergeOverrides(await getConfig(options.configPath), options);
+    logger.always("Downloading project yaml and  state from instance");
+    const state = await getState(config.statePath);
+    const { data: new_state } = await getProject(config, state.id);
+    let url = new URL(`/download/yaml?id=${state.id}`, config.endpoint);
+    const res = await fetch(url);
+    fs.writeFile(path.resolve(config.specPath), res.body);
+    fs.writeFile(path.resolve(config.statePath), new_state);
+    logger.always("Project pulled successfully");
+    process.exitCode = 0;
+    return true; 
+  } catch (error: any) {
+    throw error;
+  }
+}
+
+// Priority
```

Comment:
Comments need removing (and addressing?)

Reply:
I think this is documentation of sorts